addons/controllers/suite_test.go

```diff
@@ -240,3 +257,126 @@ var _ = AfterSuite(func() {
 	err := testEnv.Stop()
 	Expect(err).ToNot(HaveOccurred())
 })
+
+// setupWebhooks configures certs/keys and sets up the webhooks with mgr
+func setupWebhooks(ctx context.Context, mgr manager.Manager, setupLog logr.Logger) {
+	var (
+		f   *os.File
+		err error
+	)
+	// Setup TLS certs and keys
+	setupCertsAndKeysForWebhooks()
+
+	// Apply the webhooks configuration
+	webhooksFile := "webhooks/manifests.yaml"
+	f, err = os.Open(webhooksFile)
+	Expect(err).ToNot(HaveOccurred())
+	defer f.Close()
+	err = testutil.CreateResources(f, cfg, dynamicClient)
+	Expect(err).ToNot(HaveOccurred())
+
+	// Update the caBundle in validating and mutating webhooks
+	k8sConfig := mgr.GetConfig()
+	client, err := kubernetes.NewForConfig(k8sConfig)
+	validatingWebhookConf, err := client.AdmissionregistrationV1().ValidatingWebhookConfigurations().
+		Get(ctx, "validating-webhook-configuration", metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	for idx := range validatingWebhookConf.Webhooks {
+		validatingWebhookConf.Webhooks[idx].ClientConfig.CABundle = caPEM.Bytes()
+	}
+	_, err = client.AdmissionregistrationV1().ValidatingWebhookConfigurations().
+		Update(ctx, validatingWebhookConf, metav1.UpdateOptions{})
+	Expect(err).ToNot(HaveOccurred())
+
+	mutatingWebhookConf, err := client.AdmissionregistrationV1().MutatingWebhookConfigurations().
+		Get(ctx, "mutating-webhook-configuration", metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	for idx := range mutatingWebhookConf.Webhooks {
+		mutatingWebhookConf.Webhooks[idx].ClientConfig.CABundle = caPEM.Bytes()
+	}
+	_, err = client.AdmissionregistrationV1().MutatingWebhookConfigurations().
+		Update(ctx, mutatingWebhookConf, metav1.UpdateOptions{})
+	Expect(err).ToNot(HaveOccurred())
+
+	// Configure and start the webhook server
+	wh := mgr.GetWebhookServer()
+	wh.CertDir = "/tmp/k8s-webhook-server/serving-certs"
```

Comment:
no need. this is the default. btw the webhook server gets started the first time you run 
wh := mgr.GetWebhookServer() and the correct way to set the certDir is to do it in test_suite.go when you start the manager with the mgr.options { certDir: "whatever path you want"} in the test_suite.go file.